include/gamma.cuh

```diff
@@ -291,4 +291,124 @@ namespace quda {
     inline constexpr int Dir() const { return dir;  }
   };
 
+  // list of specialized structures used in the contraction kernels:
+  static constexpr int nspin = 4;
```

Comment:
I think we should get rid of this define and just have the hard-coded magic "4" in the constexpr functions below, I don't like the risk of having a variable called `nspin` baked to be one specific thing or another.

Reply:
removed in 9dfbe57a5f2b5b5c98d76bf22a24137bb848816f